groups/sig-release/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-release-leads
+  - release-engineering-approvers
```

Comment:
/hold

I don't want /approve rights in subdirectories until we have, at minimum, ensured that subdirectories aren't going to create arbitrary new groups.